amenities.mss

```diff
@@ -965,6 +1011,8 @@
   [feature = 'shop_bakery'],
   [feature = 'shop_convenience'],
   [feature = 'shop_convenience;gas'],
+  [feature = 'shop_gas'],
+  [feature = 'amenity_fuel'],
```

Comment:
I think we do not want the name of gas / fuel being displayed.

These are usually complex objects in OSM, and I think interesting names are already rendered most of the time, see https://www.cyclosm.org/#map=20/45.89039/4.44568/cyclosm.